packages/core/repl/native-functions/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './help-repl-fn';
+export * from './get-relp-fn';
```

Comment:
I think there is a minor typo here 😄
```suggestion
export * from './get-repl-fn';
```

Reply:
yeah, my bad. I fixed them in #9720